spark/src/main/kotlin/com/adevinta/spark/components/textfields/MultilineTextField.kt

```diff
@@ -114,16 +116,32 @@ public fun MultilineTextField(
     placeholder: String? = null,
     helper: String? = null,
     counter: TextFieldCharacterCounter? = null,
-    leadingIcon: @Composable (() -> Unit)? = null,
-    isError: Boolean = false,
-    error: String? = null,
+    leadingContent: @Composable (() -> Unit)? = null,
```

Comment:
is leading content is supposed to be other thing than an Icon? Maybe we should restrict the API?

Reply:
Yeah it can be a prefix/dropdown and any other plugin